src/Rewiews/style.scss

```diff
@@ -0,0 +1,82 @@
+@import './_variables.scss';
+
+.wrapper-rewiews {
+  background-image: url('../image/bg_slider.png');
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-attachment: fixed;
+}
+
+.Rewiews {
+  background-image: url('../image/bg6_slider.png');
+  width: 100%;
+  background-position: center;
+  text-align: center;
+  background-repeat: no-repeat;
+  margin-left: 20%;
+
+  padding-top: 130px;
+  padding-bottom: 100px;
+
+}
+
+.rewiew {
+  padding-top: 78px;
+  padding-bottom: 100px;
+
+  color: $white;
```

Comment:
u missed some code because of empty lines?

Reply:
changed